src/index.ts

```diff
@@ -18,29 +15,68 @@ if (!setConfig(appArgs)) {
   process.exit(0);
 }
 
-createServer((req: IncomingMessage, res: ServerResponse) => {
-  const chunks: Buffer[] = [];
-  let body: Record<string, unknown> | null = null;
-  logInfo('New connection');
-  
-  req.on('data', (data) => {
-    chunks.push(data);
-  }).on('end', () => {
-    const rawBody = Buffer.concat(chunks).toString();
-    body = JSON.parse(rawBody);
-    logInfo(JSON.stringify(body));
-    res.writeHead(200, { 'Content-Type': 'application/json' });
-    res.end(rawBody);
-  });
-}).listen(getConnectionPort(), () => logInfo(`Server listen on the ${getConnectionPort()} port`));
-
-process.on('uncaughtException', (err, origin) => {
-  logInfo(origin);
-  logError(err.message);
+const app = express();
+
+function requestLogger(request: any, response: any, next: any) {
+  const { route: { path }, method, body, query, headers } = request;
+  logInfo( `request: ${path} ${method} ${JSON.stringify(body)} ${JSON.stringify(query)} ${JSON.stringify(headers)}`);
+  next();
+}
+
+function errorHandler(error: any, request: any, response: any, next: any) {
+  console.error(`Error: ${error.message}`);
+  response.status(500).send({ error: STATUS_CODES[500] });
+  next();
```

Comment:
Does it make sense to call `next` in errorHandler ?

Reply:
you are right, this is the last middleware function and there is no need to call "next"